lib/rules/function-calc-no-unspaced-operator/index.mjs

```diff
@@ -142,7 +152,7 @@ const rule = (primary, _secondaryOptions, context) => {
 						return true;
 					}
 
-					if (currentNode.type === 'function') {
+					if (isFunction(currentNode)) {
```

Comment:
I don't know why Codecov is complaining.
The util does the exact same thing.

Reply:
I'm fixing Codecov upload error with #7622.

Reply:
The Codecov problem is resolved. Can you merge `main` into this PR branch?

Reply:
> Maybe we can refactor this file.

Not in this PR.

> The Codecov problem is resolved.

Thanks.

Reply:
> Maybe we can refactor this file.

Not in this PR.

> The Codecov problem is resolved.

Are you sure? Should I simply revert and move on instead?

Reply:
> Are you sure? Should I simply revert and move on instead?

Sorry, I meant the upload problem. This file's uncovered problem has already occurred on the `main` branch. See:

https://app.codecov.io/gh/stylelint/stylelint/blob/main/lib%2Frules%2Ffunction-calc-no-unspaced-operator%2Findex.mjs

In this PR, we can just ignore it. And, in another PR, we can refactor this file and improve the coverage.

Reply:
It would be greatly appreciated if you could work on the refactoring in another PR.